app/assets/javascripts/discourse/app/components/composer-body.js

```diff
@@ -207,11 +233,18 @@ export default Component.extend(KeyEnterEscape, {
 
   willDestroyElement() {
     this._super(...arguments);
+
     this.appEvents.off("composer:resize", this, this.resize);
     if (this._visualViewportResizing()) {
       window.visualViewport.removeEventListener("resize", this.viewportResize);
     }
 
+    START_DRAG_EVENTS.forEach((startDragEvent) => {
+      this.element
+        ?.querySelector(".grippie")
```

Comment:
Minor: are there any cases where `this.element` is falsy in `willDestroyElement()`?

Reply:
Yeh...... I tend to be over paranoid this we have ?.. I guess it's not needed indeed